modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/ObjectEntryManagerImpl.java

```diff
@@ -527,15 +636,26 @@ private Map<String, Serializable> _toObjectValues(
 	@Reference
 	private GroupLocalService _groupLocalService;
 
+	private boolean _hasPreventTypeRelationship;
+
 	@Reference
 	private ObjectEntryDTOConverter _objectEntryDTOConverter;
 
+	@Reference
+	private ObjectEntryLocalService _objectEntryLocalService;
+
 	@Reference
 	private ObjectEntryService _objectEntryService;
 
 	@Reference
 	private ObjectFieldLocalService _objectFieldLocalService;
 
+	@Reference
+	private ObjectRelationshipLocalService _objectRelationshipLocalService;
```

Comment:
Hey @p-albuquerque, could you try to use ObjectRelationshipService instead of ObjectRelationshipLocalService?
This way, we can use permissions to authorize the call to ObjectRelationshipLocalService methods.